ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionLayerManagerStub.java

```diff
@@ -14,25 +14,43 @@
 package tech.pegasys.teku.ethereum.executionlayer;
 
 import java.util.Optional;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
 import tech.pegasys.teku.infrastructure.time.TimeProvider;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 import tech.pegasys.teku.spec.Spec;
+import tech.pegasys.teku.spec.datastructures.execution.ExecutionPayloadContext;
+import tech.pegasys.teku.spec.datastructures.execution.ExecutionPayloadHeader;
+import tech.pegasys.teku.spec.datastructures.state.beaconstate.BeaconState;
 import tech.pegasys.teku.spec.executionlayer.ExecutionLayerChannelStub;
 
 public class ExecutionLayerManagerStub extends ExecutionLayerChannelStub
     implements ExecutionLayerManager {
+  private static final Logger LOG = LogManager.getLogger();
+
+  private final BuilderCircuitBreaker builderCircuitBreaker;
 
   public ExecutionLayerManagerStub(
       Spec spec,
       TimeProvider timeProvider,
       boolean enableTransitionEmulation,
-      final Optional<Bytes32> terminalBlockHashInTTDMode) {
+      final Optional<Bytes32> terminalBlockHashInTTDMode,
+      final BuilderCircuitBreaker builderCircuitBreaker) {
     super(spec, timeProvider, enableTransitionEmulation, terminalBlockHashInTTDMode);
+    this.builderCircuitBreaker = builderCircuitBreaker;
   }
 
   @Override
   public void onSlot(UInt64 slot) {
     // NOOP
   }
+
+  @Override
+  public SafeFuture<ExecutionPayloadHeader> builderGetHeader(
+      ExecutionPayloadContext executionPayloadContext, BeaconState state) {
+    LOG.info("Builder Circuit Breaker isEngaged: " + builderCircuitBreaker.isEngaged(state));
```

Comment:
do we really want to see this a lot?

Reply:
we shouldn't, it's when we activate the stub. So it is voluntarily verbose.